packages/ui/src/ui/containers/ManageTokens/ManageTokensPasswordModal/password-strategies.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
```

Comment:
It also may work without this file

Reply:
From https://github.com/ytsaurus/ytsaurus-ui/pull/980#discussion_r1944671784
```
I would prefer to combine sha256 and calling login in a single method
```
It is a good reason to leave the file